js/main.js

```diff
@@ -27,6 +29,97 @@ Site = {
   },
 };
 
+Site.Sort = {
+  siteUrl: window.location.origin + window.location.pathname,
```

Comment:
if this is only psuedo routing then I think we can use #hashbangs rather than query strings as it looks prettier. Or maybe we just use a & not a , ?